src/IndoorAtlas/IndoorAtlasLocationController.cpp

```diff
@@ -10,7 +10,7 @@ namespace ExampleApp
 {
     namespace IndoorAtlas
     {
-        IndoorAtlasLocationController::IndoorAtlasLocationController(IndoorAtlasLocationManager& locationManager,
+        IndoorAtlasLocationController::IndoorAtlasLocationController(View::IIndoorAtlasLocationManager& locationManager,
```

Comment:
This is a concurrency smell. This type takes two dependencies, one is in a View:: namespace (indicating affinity to the UI thread) and a second dependency in the SdkModel:: namespace (indicating affinity to the native thread). Could you please review whether the IIndoorAtlasLocationManager contract is ever called from the UI thread, suggest adding thread-assertions on Android. 

Having white-box reviewed this, I believe that IIndoorAtlasLocationManager is not called from the UI thread, so it is probably worth moving this types, and any other types that have native-thread affinity to SdkModel::